tests/Oryx.Integration.Tests/Python/PythonEndToEndTests.cs

```diff
@@ -66,6 +66,52 @@ public async Task CanBuildAndRun_Tweeter3AppAsync()
                 });
         }
 
+        [Fact]
+        [Trait("category", "python-3.11")]
```

Comment:
For the sake of getting something out quickly i'm fine with leaving this for a later date, but could we perhaps parameterize this testing method and invoke the django regex tests for the other python versions that had this same bug? E.g. adding some parameters to this method to have it invoked in another method like

```
[Fact]
[Trait("category", "python-3.10")]
[Trait("build-image", "github-actions-debian-bullseye")]
public async Task CanBuildAndRun_DjangoRegex_310()
{
    await this.CanBuildAndRun_DjangoRegex(PythonVersions.Python310Version, "3.10");
}
```

Reply:
Sure thing!